Makefile

```diff
@@ -25,33 +25,33 @@ style:
 	
 # Run tests for the library
 test:
-	python -m pytest -s -v ./tests/ --ignore=./tests/test_examples.py
+	python -m pytest -s -v ./tests/ --ignore=./tests/test_examples.py --durations=0
```

Comment:
I wouldn't put the `--durations=0` in the MakeFile as it's a bit spammy if a user executes the tests locally. I'd just add it to the command running the tests (which would be different from the make command).

Reply:
Will do so in a follow-up PR for the CI